c2_components/Android.mk

```diff
@@ -17,7 +17,7 @@ define build_components
       frameworks/native/libs/ui/include \
       frameworks/av/media/codec2/sfplugin/utils
 
-  LOCAL_CFLAGS := $$(MFX_C2_CFLAGS)
+  LOCAL_CFLAGS := $$(MFX_C2_CFLAGS)  -DONEVPL_EXPERIMENTAL
```

Comment:
it's better to add a comment to tell us why you added this.

Reply:
done